src/utils/components/AddBootableVolumeModal/utils/utils.ts

```diff
@@ -29,23 +43,32 @@ export const createBootableVolume =
       storageClassName,
       uploadFile,
     } = bootableVolume || {};
+
+    let newDataSource: V1beta1DataSource;
```

Comment:
no need for a let here , there is 2 places u are using it, one is scoped, u can declare it at the scope level at line 57 and also in line 97

Reply:
thanks